modules/nw-sriov-ibnetwork-object.adoc

```diff
@@ -36,4 +36,4 @@ spec:
 
 <6> Optional: The link state of virtual function (VF). Allowed values are `enable`, `disable` and `auto`.
 
-<7> Optional: The capabilities to configure for this network. You can specify `"{ "ips": true }"` to enable IP address support or `"{ "infinibandGUID": true }"` to enable IB Global Unique Identifier (GUID) support.
+<7> Optional: The capabilities to configure for this network. You can specify '{ "ips": true }' to enable IP address support or '{ "infinibandGUID": true }' to enable IB Global Unique Identifier (GUID) support.
```

Comment:
If you're trying to make these values still appear as monospace, I think they're failing to appear that way in the rendered doc because you're using an apostrophe instead of a backtick:
```suggestion
<7> Optional: The capabilities to configure for this network. You can specify `{ "ips": true }` to enable IP address support or `{ "infinibandGUID": true }` to enable IB Global Unique Identifier (GUID) support.
```

Reply:
I think this is a little more complicated than just adding backticks. Those need to be single `'` I believe. I'll check with SME and QE, but good catch either way.